modules/rosa-required-aws-service-quotas.adoc

```diff
@@ -18,71 +18,88 @@ If you need to modify or increase a specific quota, see Amazon's documentation o
 
 [options="header"]
 |===
-|Quota name |Service code |Quota code| Minimum required value | Recommended value
+|Quota name |Service code |Quota code| Default | Recommended value | Description
 
-|Number of EIPs - VPC EIPs
+|EC2-VPC Elastic IPs
 |ec2
 |L-0263D0A3
 |5
 |5
+| The maximum number of Elastic IP addresses that you can allocate for EC2-VPC in this Region.
 
 |Running On-Demand Standard (A, C, D, H, I, M, R, T, Z) instances
 |ec2
 |L-1216C47A
 |100
 |100
+| Maximum number of vCPUs assigned to the Running On-Demand Standard (A, C, D, H, I, M, R, T, Z) instances.
+
+The default value of 5 vCPUs is not sufficient to create ROSA clusters. ROSA has a minimum requirement of 100 vCPUs for cluster creation.
 
 |VPCs per Region
 |vpc
 |L-F678F1CE
 |5
 |5
+| The maximum number of VPCs per Region. This quota is directly tied to the maximum number of internet gateways per Region.
 
 |Internet gateways per Region
 |vpc
 |L-A4707A72
 |5
 |5
+| The maximum number of internet gateways per Region. This quota is directly tied to the maximum number of VPCs per Region. To increase this quota, increase the number of VPCs per Region.
 
 |Network interfaces per Region
 |vpc
 |L-DF5E4CA3
 |5,000
 |5,000
+| The maximum number of network interfaces per Region.
 
-|General Purpose SSD (gp2) volume storage
+|Storage for General Purpose SSD (gp2) volume storage in TiB
```

Comment:
```suggestion
|Storage for General Purpose SSD (gp3) volume storage in TiB
```

We use gp3 now! Updated the quota code as well

Reply:
Hey @mjlshen! I see [on the AWS docs site](https://docs.aws.amazon.com/ROSA/latest/userguide/service-quotas-rosa.html) they still use GP2. The original ask for the PR was to match their documentation. Would I go with updating our information instead?

Reply:
Gotcha, maybe this gp3 change can be in a future PR if that makes more sense. Cluster installs today definitely use gp3, so seems like we need to get both sides updated later!

Reply:
@mjlshen, nope, I was wrong! I spoke with the AWS writer, and these changes are actually correct, and they'll update the AWS documentation as well. Great catch!